unittests/unittest_poller.cpp

```diff
@@ -103,6 +103,7 @@ void wait_in_events (test_events_t &events_)
 {
     void *watch = zmq_stopwatch_start ();
     while (events_.in_events.get () < 1) {
+        msleep (1);
```

Comment:
If this change introduces a regression, that needs to be fixed by artificial sleeps, then it's not good I'm afraid. If it's just simplifying the internal details it should not have an effect on usage.

Reply:
The changes I made are in `socket_poller_t` but the tests that failed is for `poller_t` which I don't any code from the former. So I don't see how the changes I made are correlated to these failure (maybe a random unrelated issue).